.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Changes
+
+[Describe changes here]
+
+## Screenshots
+
+[Add screenshots or GIFs here]
+
+## Checklist
+
+- [ ] if adds a new page, adds accessibility tests
+- [ ] if adds a new page or new interaction, adds e2e test
+- [ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc](https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)
```

Comment:
can update this to "add a new interaction" b/c new pages get their page path automatically logged to GA

Reply:
wait i thought we had a custom trigger for this specificially because it was not happening automatically: 
```
  useEffect(() => {
    logPageView();
  }, []);
```

Reply:
resolved in slack, this is indeed needed!